README.rst

```diff
@@ -63,13 +65,27 @@ Settings list
 -------------
 All settings are contained in a ``PUSH_NOTIFICATIONS_SETTINGS`` dict.
 
-In order to use GCM, you are required to include ``GCM_API_KEY``.
-For APNS, you are required to include ``APNS_CERTIFICATE``.
+In order to use GCM, you are required to include one of ``GCM_API_KEY``, ``GCM_API_KEYS``, or ``GCM_API_KEYS_MODEL``.
+For APNS, you are required to include ``APNS_CERTIFICATE``, ``APNS_CERTIFICATES``, or ``APNS_CERTIFICATES_MODEL``.
 
 - ``APNS_CERTIFICATE``: Absolute path to your APNS certificate file. Certificates with passphrases are not supported.
+- ``APNS_CERTIFICATES``: A dictionary reflecting separate application IDs to separate APNS certificate files.
+- ``APNS_CERTIFICATES_MODEL``: A dictionary containing description of a database model to reflect
```

Comment:
We have a little different usecase, when our 3rd-party partners use API authenticated by OAuth2, to access to our site, so we need as minimum OAuth2 private/public key pairs specific for each partner identified for each Application instance. So we already have some extension for the Application model here. Moreover, we are using ready-to-use django application to implement the OAuth2 authentication server, which uses it's own separate Application model. It is much more difficult to unite the both Application models if they have a concrete implementation in django application, isn't it? So as minimum one of them should be abstract or event customizable.